src/functions-and-arrays.js

```diff
@@ -1,41 +1,141 @@
 // Iteration #1: Find the maximum
-function maxOfTwoNumbers() {}
-
+function maxOfTwoNumbers(num1, num2) {
+
+  if(num1>num2) {
+    return num1
+  }
+ 
+else {
+  return num2
+}
+}
 
 
 // Iteration #2: Find longest word
 const words = ['mystery', 'brother', 'aviator', 'crocodile', 'pearl', 'orchard', 'crackpot'];
 
-function findLongestWord() {}
+function findLongestWord(names) {
+let bigName =""
 
+if (names.length === 0) {
+  return null
+}
+
+for(let name of names) {
+  if( name.length > bigName.length) {
+ bigName = name
+  }
+}
+return bigName
+}
 
 
 // Iteration #3: Calculate the sum
 const numbers = [6, 12, 1, 18, 13, 16, 2, 1, 8, 10];
 
-function sumNumbers() {}
+function sumNumbers(numArr) {
+  let sum = 0
+
+if(numArr.length===0){
+  sum=0
```

Comment:
I think the LAB asked you to return `null` if the condition is met

here you are reassigning the variable sum to the value 0 which is already the value assigned to sum